### PR TITLE
Display APYs for past rounds

### LINKED ISSUE #430: Expand the Round History table to report 2 Columns - Passive & Active APY
### Problem:
Users are going to want to know about past performance. Further, how the current Active APY is counting out.

Add historical APY values to the Round History below. This table is designed to serve this purpose. The reward components at the top may serve other value. Let's try to leverage the table and learn from the rewards-page APY update.

##### Macro: Lean into it
I'd like to lean into it because this is the mechanism of Active Rewards & DF. It changes all the time. It's useful to know how it's evolving. I would actually like to start thinking about how we can better represent the fact that rounds, sampling, and the funding distribution evolves over the week. It's an awesome system & mechanism. Like a pair of lungs, or a tractor harvesting the field, DF is a living-breathing thing. Let's lean into this feature and figure out how to increase intuition around it.

### DoD:
- [ ] 2 columns were added to Round History table.
- [ ] Column #1 - Average Passive APY / User Passive APY
- [ ] Column #2 - Average Active APY / User Active APY
- [ ] Users are able to see the historical passive & active APYs, for the avg vs. their own performance
- [ ] Wherever there is no data. Simply display: `not available` or `0.00%`

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #430.

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
df-sql endpoint is configured to point to mainnet data to get some more accurate data for the APYs.
Passive rewards are 0 for now in df-sql, need to be calculated.

**Comment by idiom-bytes** [comment]
I have updated the headers and added tooltips. LGTM.
Screenshot from 2023-03-28 15-38-34

Prod is still reporting 0.
But, we should be able to merge this without deploying to prod.
Might look into Vercel.

**Comment by idiom-bytes** [comment]
Fixed conflicts with main.

**Review by idiom-bytes — APPROVED** [review]
LGTM

**Review by idiom-bytes — APPROVED** [review]
LGTM

**Comment by KatunaNorbert** [comment]
@idiom-bytes Let's wait for production to be ready before we merge this one

**Review by jamiehewitt15 — APPROVED** [review]
Looks good

File: src/utils/rewards.js
Comment (KatunaNorbert) [reply src/utils/rewards.js:147]: do we have that data available?

**Pull request merged**
